pagopa/pn-frontend
# Pec addtion when sercq send is enabled as default address - pn-12537

## Short description
Manage pec addition when sercq send is enabled as default address

## List of changes proposed in this pull request
- Updated LegalContacts component
- Updated PecContactItem
- Updated SercqContacItem
- Updated tests

## How to test
Login with pf/pg -> enable sercq send as default address -> check that the view change accorndly -> add pec -> check that everything is ok
